lib/ledger-node-operations.js

```diff
@@ -7,6 +7,34 @@
 
 const async = require('async');
 const validator = require('./validator');
+const bedrock = require('bedrock');
+const {config: {constants}, util: {BedrockError}} = bedrock;
```

Comment:
I think this could just be `= require('bedrock');` as I don't think bedrock is used anywhere.  Solves the alphabetical order issue as well.